### ISSUE TITLE
Category / Product listing sort arrow does not change

### ISSUE THREAD

**Issue by BenGlassAtlas** [SnowdogApps/magento2-theme-blank-sass#162]
The sorting arrow on category / product listing pages does not change when the products have been sorted.

### Descending
![desc](https://user-images.githubusercontent.com/8895777/31484078-c355aea6-af26-11e7-8045-3efdc5c6fe5d.png)

### Ascending
![asc](https://user-images.githubusercontent.com/8895777/31484084-c695a42c-af26-11e7-8abc-77b3715f8c20.png)

It is caused by an incorrect class:

```scss
vendor/snowdog/theme-blank-sass/Magento_Catalog/styles/module/_toolbar.scss:74

.sorter.sort-desc {
    &:before {
        content: $icon-arrow-down;
    }
}
```

Should be 

```scss
.sorter-action.sort-desc {
    &:before {
        content: $icon-arrow-down;
    }
}
```

I will create a pull request for this.

**Note, it looks like this issue IS in the Magento blank theme - https://github.com/magento/magento2/blob/9c14af7a97bcf338662de365d2613c7ca9dbf8c6/app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/module/_toolbar.less#L84**

**Comment by BenGlassAtlas** [comment]
Pull request created - https://github.com/SnowdogApps/magento2-theme-blank-sass/pull/163

**Comment by Igloczek** [comment]
Thanks for te contribution, but you have to wait till #161 will be finished and I release 1.0.0, b/c it's a full rewrite, so these changes might not be necessary anymore.

**Comment by BenGlassAtlas** [comment]
No problem, it's here just in case.

**Comment by Januszpl** [comment]
@BenSpace48 as we would like to keep as much as we can closely with less blank the issue was due lack of space in less blank (fixed here https://github.com/magento/magento2/commit/5c7ea5ebf18fa3e25d10cf506a348187b472384d)

Can you change to
```
.sorter {
    .sort-desc {
        &:before {
            content: $icon-arrow-down;
        }
    }
}
```

**Comment by BenGlassAtlas** [comment]
Updated the PR, thanks @Januszpl

**Comment by Igloczek** [comment]
PR merged, will be released with 1.1.0.
